### ISSUE TITLE
Effective Range

### ISSUE THREAD

**Issue by Silverfang22** [JKGDevs/JediKnightGalaxies#209]
When a blaster shot goes past its maximum range instead of disappearing immediately it should start to expontentially lose damage over distance, possibly also use a smaller projectile effect.

Additionally, when it plays the "Death efx" of the projectile, that effect should fly at the same velocity the shot was going.

**Comment by eezstreet** [comment]
I'm going to move this to Unknown for right now because I'm not sure this is very high priority, it's a very minor consideration in the majority of maps we have now.

**Comment by Silverfang22** [comment]
Is now a thing per @DarthFutuza with f2b0db15da3216b3ecc4863a4eed94aca702e71e

Still may need Death efx to continue flying when the projectile dies.

**Comment by DarthFutuza** [comment]
Yeah I didn't do anything with efx yet, but they should probably get weaker or fade or something as damage approaches 0.
